internal/provider/serverless_cluster.go

```diff
@@ -54,19 +57,79 @@ func ResourceServerlessCluster() *schema.Resource {
 				// always returns a non-nil error.
 				// ForceNew: true,
 				Type: schema.TypeString,
+				// being ForceNew; Scylla Cloud API does not allow for
+				// updating existing clusters, thus update the implementation
+				// always returns a non-nil error.
+				// ForceNew:    true,
 			},
 			"free_tier": {
 				Description: "Whether a cluster is in a free tier",
-				Computed:    true,
+				Default:     true,
+				Type:        schema.TypeBool,
+				Optional:    true,
+				// being ForceNew; Scylla Cloud API does not allow for
+				// updating existing clusters, thus update the implementation
+				// always returns a non-nil error.
+				// ForceNew:    true,
```

Comment:
let's uncomment this (commented L63 is enough)

Reply:
done